### ISSUE TITLE
failed: open C:\Users\ADMINI~1\AppData\Local\Temp\astibundler\bind\resources\app\static\css\base.css: Access is denied

### ISSUE THREAD

**Issue by zichen8552** [asticode/go-astilectron-demo#79]
windows can't use this :astilectron-bundler

**Comment by asticode** [comment]
It seems file is in the administrator folder, are you using the administrator as well to open your app ?

**Comment by zichen8552** [comment]
Yes, I am operating under the administrator account, I will study it again

**Comment by lobsterb** [comment]
I have the same problem, how to fix

**Comment by ibbo** [comment]
Same problem here, except I'm not accessing the Administrator folder, it's in my normal user folder. Fails in a normal shell and in a shell running as Administrator too.
